# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/DCLayerTree.cpp
# Commit: 017a5fb444d4
# Full Hash: 017a5fb444d4f5e1eb1772cd89db2244afa8cce4
# Author: Cristina Horotan <chorotan@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1945683
# File Overlap Count: 1
# Description:
#   Revert "Bug 1945683 - Enable layer compositor on Windows in Nightly r=gfx-reviewers,lsalzman" for causing reftest failures
#   
#   This reverts commit 5daef99508f341552c9b2441b46c50f5bec3a22b.
# ==============================================================================

diff -r 625988823050 -r 017a5fb444d4 gfx/webrender_bindings/DCLayerTree.cpp
--- a/gfx/webrender_bindings/DCLayerTree.cpp	Fri Jun 13 07:03:22 2025 +0300
+++ b/gfx/webrender_bindings/DCLayerTree.cpp	Fri Jun 13 08:57:44 2025 +0300
@@ -547,8 +547,13 @@
 }
 
 bool DCLayerTree::UseLayerCompositor() const {
+// Only allow the layer compositor in nightly builds, for now.
+#ifdef NIGHTLY_BUILD
   return UseNativeCompositor() &&
          StaticPrefs::gfx_webrender_layer_compositor_AtStartup();
+#else
+  return false;
+#endif
 }
 
 void DCLayerTree::DisableNativeCompositor() {